libkbfs/config_local.go

```diff
@@ -613,6 +613,12 @@ func (c *ConfigLocal) RekeyWithPromptWaitTime() time.Duration {
 	return c.rwpWaitTime
 }
 
+// SetRekeyWithPromptWaitTime implements the Config interface for
+// ConfigLocal.
+func (c *ConfigLocal) SetRekeyWithPromptWaitTime(d time.Duration) {
+	c.rwpWaitTime = d
```

Comment:
Should probably use `c.lock` both here and in the getter, since it's settable now after the config is made.